components/identities/availableIdentities.tsx

```diff
@@ -1,272 +1,225 @@
+                              >
+                                 {domain.domain ? domain.domain : domain.id}
+                              </button>
+                           ))}
+                        </div>
+                        <button
+                           className="  w-full justify-center text-center items-center font-quickZap font-normal min-h-[40px] py-5 flex gap-2 bg-white rounded-b-2xl"
+                           onClick={address ? () => mint() : () => setShowWalletConnectModal(true)}
+                        >
+                           <FaPlus />
+                           ADD IDENTITIES
+                        </button>
+                     </div>
+                  </div>
+                  <div className={` flex justify-center items-center ${styles.CardContainer}`}>
+                     <IdentityCard identity={identity} tokenId={tokenId} isOwner={isOwner} onPPClick={() => setIsUpdatingPp(true)} ppImageUrl={ppImageUrl} />
+                  </div>
+                  <div className=" min-w-[280px]">
+                     {hideActions ? (
+                        <IdentityActions isOwner={isOwner} tokenId={tokenId} isIdentityADomain={isIdentityADomain} identity={identity}  />
+                     ) : (
+                        minting && <IdentityActionsSkeleton />
+                     )}
                   </div>
-                  <IdentityWarnings
-                    isIdentityADomain={isIdentityADomain}
-                    identity={identity}
-                  />
-                </>
-              </div>
-            </div>
-          </div>
-        ) : (
-          <UpdateProfilePic
-            tokenId={tokenId}
-            back={() => setIsUpdatingPp(false)}
-            openTxModal={() => setIsTxModalOpen(true)}
-            setPfpTxHash={setPpTxHash}
-          />
-        )}
-      </div>
-      <TxConfirmationModal
-        txHash={ppTxHash}
-        isTxModalOpen={isTxModalOpen}
-        closeModal={() => setIsTxModalOpen(false)}
-        title="Your new profile picture is being set !"
-      />
-      <WalletConnect
-        closeModal={() => setShowWalletConnectModal(false)}
-        open={showWalletConnectModal}
-        connectors={connectors as Connector[]}
-        connectWallet={connectWallet}
-      />
-    </>
-  );
+               </div>
+            ) : (
+               <UpdateProfilePic tokenId={tokenId} back={() => setIsUpdatingPp(false)} openTxModal={() => setIsTxModalOpen(true)} setPfpTxHash={setPpTxHash} />
+            )}
+         </div>
+         <TxConfirmationModal txHash={ppTxHash} isTxModalOpen={isTxModalOpen} closeModal={() => setIsTxModalOpen(false)} title="Your new profile picture is being set !" />
+         <WalletConnect closeModal={() => setShowWalletConnectModal(false)} open={showWalletConnectModal} connectors={connectors as Connector[]} connectWallet={connectWallet} />
+      </>
+   );
 };
```

Comment:
It should be the opposite: `!hideActions`, or rename it `showActions`. Because currently, when hideActions === true, you show them, it's the opposite of what it means in english